packages/web-app-files/src/components/SideBar/Links/PublicLinks/LinkInfo.vue

```diff
@@ -143,7 +143,7 @@ export default {
       return this.$gettextInterpolate(
         translated,
         {
-          linkName: link.name
+          linkName: link.name ? link.name : ''
```

Comment:
In the case of a link that has no name, the message will be set with the undefined value.